gdx/src/com/badlogic/gdx/net/HttpRequestBuilder.java

```diff
@@ -50,7 +49,7 @@ public HttpRequestBuilder newRequest () {
 			throw new IllegalStateException("A new request has already been started. Call HttpRequestBuilder.build() first.");
 		}
 
-		httpRequest = Pools.obtain(HttpRequest.class);
+		httpRequest = new HttpRequest();
```

Comment:
Giving up here? ;)

Reply:
I totally forgot to mention it in the PR description, but I'm not sure whether pooling makes sense here?
The object is never freed, the object doesn't have a clear lifecycle and the allocation seems not really happen often?
It also makes the code not thread safe, without it needing to.